src/AspNet.Security.OAuth.XiaoMi/XiaoMiAuthenticationConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿/*
```

Comment:
I can't comment on file-names in git diffs, but you also need to fix the casing of the directory and file names.

Reply:
This is why the ubuntu build is failing.

Reply:
I confirmed that it was all renamed, but nothing changed after the commit, and I don't know why, so I decided to re-commit.

![xiaomiprovider](https://user-images.githubusercontent.com/35516482/167168473-691cc6d9-674f-4ceb-80f1-dad876546a54.png)

Reply:
Windows is a case-insensitive file system, so just changing the case in the file names won't make them change in git.

You can achieve a rename either by renaming to _something else_ and then back to the intended casing, e.g. `XiaoMiAuthenticationOptions2` -> `XiaomiAuthenticationOptions`, or by using the `git mv` command, e.g. `git mv XiaoMiAuthenticationOptions.cs XiaomiAuthenticationOptions.cs`.

Reply:
I know the first way but don't like it, I will try the second way in the future, thanks.